chia/rpc/data_layer_rpc_api.py

```diff
@@ -36,11 +37,18 @@ def process_change(change: Dict[str, Any]) -> Dict[str, Any]:
     }
 
 
+def get_fee(config: Dict[str, Any], request: Dict[str, Any]) -> uint64:
+    fee = request.get("fee")
+    if fee is None:
+        config_fee = config.get("fee", 0)
+        return uint64(config_fee)
```

Comment:
This doesn't cover the case where the config has `None` for example from a line like `fee:` with no value I believe.

https://github.com/Chia-Network/chia-blockchain/pull/10042#discussion_r798788659